pages/reference/cli.md

```diff
@@ -2,7 +2,7 @@
 
 The balena CLI (Command-Line Interface) allows you to interact with the balenaCloud and the
 [balena API](https://www.balena.io/docs/reference/api/overview/) through a terminal window
-on Linux, MacOS or Windows. You can also write shell scripts around it, or import its Node.js
+on Linux, macOS or Windows. You can also write shell scripts around it, or import its Node.js
```

Comment:
This is odd... The website at the moment already shows the "changes" being "introduced" in this PR, at least for the CLI docs: https://www.balena.io/docs/reference/cli/

Reply:
For the supervisor as well...

Reply:
Hm, so it is! I don't know why that would be, I was under the impression this PR was required (obviously). I guess I should merge this PR to stay consistent, though presumably it does not matter too much..